src/sass/components/_feature-box.scss

```diff
@@ -61,3 +62,14 @@
     }
   }
 }
+
+
+// this is not working, but I cannot find any higher specificity selector overwriting this:
+.feature-link {
+  text-decoration: none;
+}
+
+// this is a workaround, alternatively I could use !important with class selector above
+#feature-link-1, #feature-link-2, #feature-link-3, #feature-link-4 {
```

Comment:
instead of ids you can add .feature-link:hover {} because of additional declaration done by framework:
![image](https://user-images.githubusercontent.com/2649659/47594796-46fe2400-d97d-11e8-96ab-3732d7a93e91.png)